spec/outputs/csv_spec.rb

```diff
@@ -7,18 +7,25 @@
 
   subject { described_class.new(options) }
 
-  let(:tmpfile) { Tempfile.new('logstash-spec-output-csv').path }
```

Comment:
Ah, yeah looks like the bug was that the a new `Tempfile` was created (side effect of actually making the file) but there was no reference to it because this was simply reference to the `path`. So it was free to be cleaned up. Keeping the reference to the Tempfile object should solve it.